resources/definitions/HardPrint_Hozo_iX.def.json

```diff
@@ -0,0 +1,36 @@
+{
```

Comment:
Could you change the file name, and any IDs referring to it, to be lowercase, like the rest of the printer definitions?